Sources/Filters/Texture/TextureMapToPlane/index.js

```diff
@@ -87,7 +87,7 @@ function vtkTextureMapToPlane(publicAPI, model) {
     const c1 = [m[0], m[1], m[2]];
     const c2 = [m[3], m[4], m[5]];
     const c3 = [m[6], m[7], m[8]];
-    const matrix = [c1, c2, c3];
+    const matrix = m;
```

Comment:
this new variable matrix does not seem needed anymore, you could use `m` instead